FEM/fem_ele_vec.cpp

```diff
@@ -848,26 +848,14 @@ void CFiniteElementVec::ComputeMatrix_RHS(const double fkt, const Matrix* p_D)
 	Matrix* tmp_AuxMatrix2 = AuxMatrix2;
 	Matrix* tmp_Stiffness = Stiffness;
 
-	int act_r = 0;
-#if defined(USE_PETSC) // || defined(other parallel libs)//05.3013. WW
-	act_r = act_nodes_h;
-#else
-	act_r = nnodesHQ;
-#endif
-
-	for (i = 0; i < act_r; i++)
+	for (i = 0; i < nnodesHQ; i++)
```

Comment:
There is not any bug here. Removed the ifdef is just to make the this piece of source code be simple, however a little  computation load could be raised.